packages/nushell/project.bri

```diff
@@ -50,7 +50,6 @@ export default function nushell(): std.Recipe<std.Directory> {
   const nushell = cargoBuild({
     source,
     runnable: "bin/nu",
-    dependencies: [openssl],
```

Comment:
The plugins still require the `openssl`, though. That's why this dependency is still here.